lib/appliance_console/certificate_authority.rb

```diff
@@ -108,8 +108,9 @@ def configure_api
       ).request
       if cert.complete?
         say "configuring apache to use certs"
-        FileUtils.cp("#{TEMPLATES}/etc/httpd/conf.d/cfme-https-cert.conf", "/etc/httpd/conf.d/cfme-https-cert.conf")
-        LinuxAdmin::Service.new("httpd").restart
+        FileUtils.cp("#{TEMPLATES}/etc/httpd/conf.d/cfme-https-cert.conf",
+                     Pathname.new(ENV['MIQ_APACHE_ROOT_DIR']).join("etc/httpd/conf.d/cfme-https-cert.conf").to_s)
```

Comment:
I think FileUtils.cp should handle Pathnames, so I don't believe you need the `.to_s`